packages/web-components/netlify.toml

```diff
@@ -1,4 +1,4 @@
 [build]
 
 publish = "storybook-static/"
-command = "npm run build"
+command = "npm run build.stencil && npm run build.storybook"
```

Comment:
@markacianfrani FWIW build.storybook is still there to use with the netlify build